scripts/install-tools.sh

```
@@ -12,15 +12,27 @@ else
     exit 1
 fi
 
+# Detect OS family
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
```

Shellcheck (0.11.0)

[info] Not following: /etc/os-release was not specified as input (see shellcheck -x).

(SC1091)